tsconfig-node.json

```diff
@@ -1,25 +1,9 @@
 // This tsconfig is used to build the model layer only for use in a node.js app.
 // Build with ./node_modules/.bin/tsc -p tsconfig-models.json
```

Comment:
```suggestion
// Build with ./node_modules/.bin/tsc -p tsconfig-node.json
```

Reply:
I've never properly read that comment. Good fix.